docs/reference/transform/transform-alerts.asciidoc

```diff
@@ -18,19 +18,20 @@ refer to
 
 You can create {transform} rules under **{stack-manage-app} > {rules-ui}**.
 
-. On the *Create rule* window, give a name to the rule and optionally provide 
-tags. Select the {transform} health rule type:
+. Click *Create rule* and select the {transform} health rule type.
+
+. Give a name to the rule and optionally provide tags.
+
+. Select the {transform} or {transforms} to include. You can also use a special 
+character (`*`) to apply the rule to all your {transforms}. {transforms-cap} 
+created after the rule are automatically included.
```

Comment:
"rule is" or "rules are"?

Reply:
That blurb appears as following in the preview, which I think looks right: "You can also use a special character (*) to apply the rule to all your transforms. Transforms created after the rule are automatically included."